djlsp/parser.py

```diff
@@ -157,7 +157,16 @@ def completions(self, line, character):
 
         for regex, completion in matchers:
             if match := regex.match(line_fragment):
-                return completion(match)
+                # Sort completions because some editors (Helix) will use order
+                # as is and wont use sort_text.
+                return list(
+                    sorted(
+                        completion(match),
+                        key=lambda comp: (
+                            comp.sort_text if comp.sort_text else comp.label
+                        ),
+                    )
+                )
```

Comment:
Wouldn't it be nicer if sorting the completions happens in it's own separate function?

Reply:
Nicer as in more readable? And you mean the lamba function?

Reply:
Nicer as in better readability.